### DISCUSSION TITLE
Mode Nuit pour ajouter sur nos packs STUdio ?

### DISCUSSION THREAD

**Discussion by beringcat** [marian-m12l/studio#261] [General]
bonjour, comment puis je ajouter le mode nuit à mes differents packs editables dans STUdio ? :-)

**Comment by jersou** [comment]
Bonjour, il y a une explication dans le channel discord : 
https://discord.com/channels/911349645752541244/933370265159737364

en gros, il faut que toutes les histoires aient une transition "OK" vers un noeud action, que ce noeud acton pointe vers un nouveau noeud stage, et enfin, que ce dernier pointe vers le premier noeud action du pack.

Ce mode vient d'être ajouté à https://github.com/jersou/studio-pack-generator pour info
